lms/models/assignment.py

```diff
@@ -73,7 +86,7 @@ class Assignment(CreatedUpdatedMixin, Base):
           In those cases resource_link_id will be the 1.1 value and we'll store here the 1.3 version.
     """
 
-    tool_consumer_instance_guid = sa.Column(sa.Unicode, nullable=False)
+    tool_consumer_instance_guid: Mapped[str] = mapped_column(sa.Unicode)
```

Comment:
Adapting this field to the new style.

Reply:
Yes it is, removed all of this. The one we also often use "UnicodeText" is not the default so that one can't be removed. 

For new models I reckon we always use the default.